matrix.sh

```diff
@@ -54,7 +54,13 @@ help() {
 }
 
 _curl() {
-	curl -s -H "$AUTHORIZATION" -H "User-Agent: matrix.sh/$VERSION" "$@"
+	HEADERFILE="$HOME/.matrix.sh-headers"
+	if [ -z "$(grep "$AUTHORIZATION" $HEADERFILE)" ] ; then
```

Comment:
Please quote all variables that don't need word splitting. (I know it's unlikely that $HOME has whitespace in it, but it's better to be safe than sorry.)